packages/table/src/table.tsx

```diff
@@ -41,6 +42,14 @@ import { TableBody } from "./tableBody";
 
 export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     /**
+     * If `true`, there will be a single "focused" cell at all times, 
+     * which can be used to interact with the table as though it is a
+     * spreadsheet.
+     * @default false
```

Comment:
I'm 👎 -- and this is because the focus cell is probably _not_ what most consumers of a table library are expecting. If we turn this on-by-default then upon upgrade all of our consumers will suddenly have a focus cell -- whether they want to or not.